tests/time.go

```diff
@@ -0,0 +1,13 @@
+package tests
```

Comment:
Please write these tests in [testcases](https://github.com/ferranbt/fastssz/tree/master/sszgen/testcases) instead.

Reply:
Moved.